### PR TITLE
Add option to only elaborate the base design on FPGA

### PR DESCRIPTION
**Pull request by alex96295**

### CODE REVIEW

File: target/xilinx/xilinx.mk [hunk 1]
```diff
@@ -61,8 +63,8 @@ $(mcs): $(bit)
 # Compile bitstream
 $(bit): $(ips) $(CAR_XIL_DIR)/scripts/add_sources.tcl
 	@mkdir -p $(out)
-	cd $(CAR_XIL_DIR) && $(VIVADOENV) $(VIVADO) $(VIVADOFLAGS) -source scripts/run.tcl
-	cp $(CAR_XIL_DIR)/$(PROJECT).runs/impl_1/$(PROJECT)* $(out)
+	cd $(CAR_XIL_DIR) && $(VIVADOENV) $(VIVADO) $(VIVADOFLAGS) -source scripts/run.tcl -tclargs $(ELABORATION_ONLY)
+	if [ "$ELABORATION_ONLY" -eq 0 ]; then cp $(CAR_XIL_DIR)/$(PROJECT).runs/impl_1/$(PROJECT)* $(out); fi
```
Comment (micprog) [inline target/xilinx/xilinx.mk:67]:
```suggestion
	if [ "$(ELABORATION_ONLY)" -eq 0 ]; then cp $(CAR_XIL_DIR)/$(PROJECT).runs/impl_1/$(PROJECT)* $(out); fi
```

**Comment by CyrilKoe** [comment]
Better doing like this: 
https://github.com/pulp-platform/carfield/pull/175/files#diff-8eefe046bc8ff9d46584ab747bfb48ed8bc0c5841ac5c131c52be9075391412f

**Comment by alex96295** [comment]
> Better doing like this: https://github.com/pulp-platform/carfield/pull/175/files#diff-8eefe046bc8ff9d46584ab747bfb48ed8bc0c5841ac5c131c52be9075391412f

Can you quickly hack it on this branch please? Thanks

**Comment by alex96295** [comment]
> > Better doing like this: https://github.com/pulp-platform/carfield/pull/175/files#diff-8eefe046bc8ff9d46584ab747bfb48ed8bc0c5841ac5c131c52be9075391412f
> 
> Can you quickly hack it on this branch please? Thanks

@CyrilKoe ?

**Comment by alex96295** [comment]
I guess this is integrated in #175 , so I close it

**Pull request closed without merging**